osh/hub/settings.py

```diff
@@ -188,28 +192,15 @@
     pass
 
 
-def _get_secret(name):
-    try:
-        with open(os.path.join(SECRETS_DIR, name)) as f:
-            return f.read().strip()
-    except OSError:
-        return None
-
+###############################################################################
+# Secrets
+###############################################################################
 
-BZ_API_KEY = _get_secret('bugzilla_secret')
-JIRA_API_KEY = _get_secret('jira_secret')
+from osh.hub.other.settings_util import get_secret  # noqa: E402
```

Comment:
Does it still need to be imported here?  The access to `SECRETS_DIR` from the function is removed in d5372cae14172eedc45f6aa2bea0f9c648229f64.

Reply:
Yes, otherwise the `get_secret` function would be undefined and Python Interpreter would raise `NameError`.

Reply:
Well, technically, the symbol might get imported transitively from `settings_local.py` but that is not guaranteed.

Reply:
I understand that `get_secert()` needs to be imported.  I am just asking why it cannot be imported at the top of this file without using `# noqa: E402`.

Reply:
Sorry, I have missed this reply.  I've just aggregated all secret-related stuff to one place.  Addressed in a1d083c25b781366f45fedfe6097aedb3956c385.